Ravel-Creators/Assets/RavelCreator/Scripts/Editor/CreatorTools/Assetbundles/BundleBuilder.cs

```diff
@@ -123,8 +201,7 @@ public static class BundleBuilder
 		}
 		
 		//build the actual bundle
-		var assetBundleManifest = BuildPipeline.BuildAssetBundles(path, BuildAssetBundleOptions.None,
-			BuildTarget.WebGL);
+		BuildPipeline.BuildAssetBundles(path, BuildAssetBundleOptions.None, BuildTarget.WebGL);
```

Comment:
kill is an odd name for a list, rename to something like FilesToDelete or something more descriptive